timemachine/fe/rbfe.py

```diff
@@ -617,47 +606,44 @@ def estimate_relative_free_energy_via_greedy_bisection(
 
 
 def run_vacuum(
-    mol_a,
-    mol_b,
-    core,
-    forcefield,
+    mol_a: Chem.rdchem.Mol,
+    mol_b: Chem.rdchem.Mol,
+    core: NDArray,
+    forcefield: Forcefield,
     _,
-    n_frames,
-    seed,
-    n_eq_steps=10000,
-    steps_per_frame=400,
-    n_windows=None,
-    min_cutoff=None,
+    seed: int,
+    md_params: Optional[MDParams] = None,
+    n_windows: Optional[int] = None,
+    min_cutoff: Optional[float] = None,
 ):
+    if md_params is not None and md_params.local_steps > 0:
+        md_params = replace(md_params, local_steps=0)
+        warnings.warn("Vacuum simulations don't support local steps, will use all global steps")
```

Comment:
Could this just
```suggestion
    assert md_params is not None and md_params.local_steps > 0, "Vacuum simulations don't support local steps, will use all global steps"
```
(slight preference for this; warnings can be easy to miss)

Reply:
Changed in https://github.com/proteneer/timemachine/pull/1026/commits/98a2c1d97b61405788fcee301953f778c4736450